### DISCUSSION TITLE
Modifications to the Etsy OpenAPI Specification

### DISCUSSION THREAD

**Discussion by cilerler** [etsy/open-api#1016] [Q&A]
- [ ] In the Etsy OpenAPI specification, a `"maximum": 1.79769313486e308` is set twelve times. This inclusion causes a breakdown in the code due to the presence of `e` in the value. I kindly request you to eliminate the `maximum` property from these respective objects.

- [ ] In addition to the aforementioned modification, I also request you to incorporate `"format": "int64"` into the definitions where `"type":"integer"` is currently specified. This amendment will greatly improve the value handling.

- [ ] Could you kindly incorporate the absent `edit` value into the `ShopListingWithAssociations.properties.state.enum` in the current configuration?

- [ ] Please include the attribute `"nullable": true` for the properties `adjusted_gross`, `adjusted_fees`, and `adjusted_net` in the `Payment` object.

- [ ] please update `ShopListingWithAssociations.properties.translations` 
  from
  ```json
  "translations": {
      "type": "array",
      "description": "An array of translations for the listing. Default value is an empty array.",
      "items": {
          "description": "An array of translations for the listing. Default value is an empty array.",
          "allOf": [
              {
                  "$ref": "#/components/schemas/ListingTranslation"
              }
          ]
      }
  },
  ```
  to
  ```json
  "translations": {
      "type": "object",
      "description": "A dictionary of translations for the listing. Default value is an empty dictionary.",
      "additionalProperties": {
          "$ref": "#/components/schemas/ListingTranslation"
      }
  },     
  ```

**Comment by Gareth064** [comment]
Hi @cilerler 

This discussion repo is for the community to discuss things and help each other, plus the odd Announcement from the Etsy Devs. (They used to engage regularly but not anymore)

If you have a request or issue that you want Etsy to address then you need to email them at developer@etsy.com

Regarding what you've mentioned, the one about int64 has been brought up numerous times over the past 3 years and it's never been resolved. 😞

**Comment by cilerler** [comment]
I have been specifically instructed by the Etsy Developers to make this post here thus re-opening it.
